modules/migration-mtc-release-notes-1-8-6.adoc

```diff
@@ -0,0 +1,52 @@
+
+When creating a migration plan in {mtc-short}, the UI becomes stuck on the *Persistent Volumes* and you cannot continue. This issue occurs due to a critical reconciliation failure and returns a 404 API error when you attempt to fetch the migration plan from the backend. These issues cause the migration plan to remain in a *Not Ready* state, and you are prevented from continuing. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1705[(MIG-1705)]
+
+.Migration process becomes stuck after the `StageBackup` phase
+
+When migrating a Django and PostgreSQL application, the migration becomes stuck after the `StageBackup` phase. Even though all the pods in the source namespace are healthy before the migration begins, after the migration and when terminating the pods on the source cluster, the Django pod fails with a `CrashLoopBackOff` error. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1707[(MIG-1707)]
+
+.Migration shown as succeeded despite a failed phase due to a misleading UI status
+
+After running a migration using {mtc-short}, the UI incorrectly indicates that the migration was successful, with the status shown as *Migration succeeded*. However, the Direct Volume Migration (DVM) phase failed. This misleading status appears on both the *Migration* and the *Migration Details* pages. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1711[(MIG-1711)] 
+
+.Persistent Volumes page hangs indefinitely for namespaces without PVCs
+When a migration plan includes a namespace that does not have any persistent volume claims (PVCs), the *Persistent Volumes* selection page becomes stuck with the following message shown: `Discovering persistent volumes attached to source projects...`. The page never completes loading, preventing you from proceeding with the migration. This issue has been resolved in {mtc-short} 1.8.6. link:https://issues.redhat.com/browse/MIG-1713[(MIG-1713)]
+
+[id="known-issues-1-8-6_{context}"]
+== Known issues
+
+{mtc-short} 1.8.6 has the following known issues:
+
+.Inconsistent reporting of migration failure status
+
+There is a discrepancy in the reporting of namespace migration status following a rollback and subsequent re-migration attempt when the migration plan is deliberately faulted. Although the Distributed Volume Migration (DVM) phase correctly registers a failure, this failure is not consistently reflected in the user interface (UI) or the migration plan's YAML representation. 
+
+This issue is not only limited to edge cases. In certain circumstances, such as when network restrictions are applied that cause the DVM phase to fail, the UI still reports the migration status as successful. This behavior is similar to what was previously observed in link:https://issues.redhat.com/browse/MIG-1711[MIG-1711] but occurs under different conditions. link:https://issues.redhat.com/browse/MIG-1719[(MIG-1719)] 
```

Comment:
Not sure about using the term "edge case". Not everyone will know what that means and it will translate poorly. Perhaps "unusual cases"?